src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy

```diff
@@ -221,6 +221,7 @@ class IqPolicyEvaluatorIntegrationTest
       def build = project.scheduleBuild2(0).get()
 
     then: 'an exception is thrown when getting proprietary config from IQ server'
+      //1 * iqClient.validateServerVersion(*_)
```

Comment:
Why are all these commented out (here and below)?

```
 //1 * iqClient.validateServerVersion(*_)
```

Reply:
>Why are all these commented out (here and below)?

sorry, leftover artifacts that I thought I had removed.  removed in cae09113726f9e0ff949b62b9556e64ec1702020